components/threading/channel_threads/channel_threads.stories.tsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import moment from 'moment-timezone';
+
+import {Unit} from '@formatjs/intl-relativetimeformat';
+
+import {storiesOf} from '@storybook/react';
+import {withKnobs, boolean, number, select} from '@storybook/addon-knobs';
+import {action} from '@storybook/addon-actions';
+
+import StoryGrid from 'storybook/story_grid';
+import StoryBox from 'storybook/story_box';
+
+import ThreadFooter from './thread_footer';
+
+const units: Unit[] = [
+    'second',
+    'minute',
+    'hour',
+    'day',
+    'week',
+    'month',
+    'quarter',
+    'year',
+];
+
+const users = [
+    {
+        url: '/api/v4/users/1/image?_=0',
```

Comment:
Can we change the urls to `./api/v4/users/...`, that would make it work in the local and online storybook.